src/main.rs

```diff
@@ -16,6 +16,7 @@ use image::GenericImage;
 use image::Pixel;
 use image::RgbaImage;
 use image::Rgba;
+use image::pnm::*;
```

Comment:
Please don't use *. Replace this with `use image::pnm;` and adjust code accordingly, or `use` only what you need.

Reply:
a.k.a.
```suggestion
use image::pnm::PNMSubtype;
```

Although we did this because before we had:
```rust
image::ImageOutputFormat::PNM(image::pnm::PNMSubtype::Pixmap(image::pnm::SampleEncoding::Binary))
```

But now it's just
```
image::ImageOutputFormat::PNM(image::pnm::PNMSubtype::ArbitraryMap))
```

So, there's less need for the `use` now.